examples/rules/stand_alone.py

```diff
@@ -31,12 +33,26 @@
 }
 
 
+# Checks if Fedora, if so returns version, if not raises SkipComponent
+@component(RedhatRelease)
+class Is_Fedora(object):
+    def __init__(self, rhel):
+        self.content = []
+        if "Fedora" in rhel.product:
+            self.rhel_version = rhel.version
+        else:
+            raise SkipComponent('Not Fedora')
+
+
 class Specs(SpecSet):
     """ Datasources for collection from local host """
     hosts = simple_file("/etc/hosts")
 
 
-@parser(Specs.hosts)
+# If this runs on a server that is not Fedora or with an archive that
+# is not from a Fedora server this parser will not fire since the
+# Is_Fedora component raises SkipComponent if not Fedora
+@parser(Specs.hosts, Is_Fedora)
```

Comment:
One more question just came into mind, seems the `Requires`, `At Least One` or `Optional` feature also works to the @parser (since it's also a@ component).  Would you please add one more example for applying multiple dependencies/components to one @parser? For example, when a @parser works for both rhel-6 and rhel-7 (but except for rhel-8).

Reply:
Sure thing, I reverted stand_alone.py and created a new module named skip_component.py in the examples. It demonstrates a parser and rule that just looks for Rhel8 and then another that  parser and rule that demonstrates a combination of Rhel6 and Rhel7 dependencies.